receiver/awscontainerinsightreceiver/internal/host/nodeCapacity.go

```diff
@@ -49,11 +49,14 @@ func newNodeCapacity(logger *zap.Logger, options ...nodeCapacityOption) (nodeCap
 		opt(nc)
 	}
 
-	if _, err := nc.osLstat(hostProc); os.IsNotExist(err) {
-		return nil, err
-	}
-	if err := nc.osSetenv(goPSUtilProcDirEnv, hostProc); err != nil {
-		return nil, fmt.Errorf("NodeCapacity cannot set goPSUtilProcDirEnv to %s: %w", hostProc, err)
+	// On Windows, hostProc is empty
+	if hostProc != "" {
```

Comment:
Nit: While the current approach effectively handles the condition, a way to potentially improve readability could be to introduce a variable like `isNotWindows`.

For example:
```
var isNotWindows = hostProc != ""
if isNotWindows {
     // Existing logic...
}
```